flexmeasures/api/v3_0/assets.py

```diff
@@ -97,24 +99,33 @@ def index(
 
         This endpoint returns all accessible assets for the account of the user.
         The `account_id` query parameter can be used to list assets from a different account.
-        The `all_accessible` query parameter can be used to list all the assets accessible by the requesting user. Defaults to `false`.
+        The `all_accessible` query parameter can be used to list assets not only from the current user's account, but from all accounts the current_user has read-access to, plus all public assets. Defaults to `false`.
+
+        The endpoint supports pagination of the asset list using the `page` and `per_page` query parameters.
+            - If a `page` parameter is provided, the response will be paginated, showing a specific number of assets per page as defined by `per_page` (default is 10).
+            - If the `page` parameter is not provided, all assets are returned without pagination.
+        The response schema inspires from https://datatables.net/manual/server-side#Returned-data
+
 
         **Example response**
 
         An example of one asset being returned:
 
         .. sourcecode:: json
 
-            [
-                {
+            {
+                "data" : {
```

Comment:
This looks like a breaking change to the API. Or will the response stay the same as before in case the `page` parameter is omitted?

Reply:
No, the response is uniform, so it actually is a breaking change currently.

Reply:
There, we can in principle send the old format if no `page` is sent in. Shall I do that?